packages/client/src/arpa_reporter/components/DownloadFileButton.vue

```diff
@@ -11,6 +11,11 @@ export default {
   name: 'DownloadFileButton',
   props: {
     upload: Object
+  },
+  computed: {
```

Comment:
should this be `methods` instead?

Reply:
Unclear... I was more or less following the [`DownloadTemplateBtn.vue` component template ](https://github.com/usdigitalresponse/usdr-gost/blob/fd95ebf635a1870a7e9ec214ed0a1e165b1698e0/packages/client/src/arpa_reporter/components/DownloadTemplateBtn.vue) as a guide, which uses `computed` to set the `href` value for the rendered component.